data/korath.txt

```diff
@@ -576,8 +576,7 @@ outfitter "Korath Basics"
 	"Command Center"
 	"Small Bunk Room"
 	"Bunk Room"
-	"Fuel Pod"
```

Comment:
I would not remove the Fuel Pod.

Reply:
Why not? This new system is the Korath fuel pod.

Reply:
I would consider the pod as the "small" version of the collector, used by Korath captains who just need a touch more fuel for their purposes (and don't want to wait to collect it).

Reply:
Perhaps we should have two versions then:
* Small Fuel Pod: 10 mass, 100 capacity, .2 ramscoop; for sale
* Large Fuel Pod: 20 mass, 200 capacity, .5 ramscoop; installed in Raiders and World-Ships, not for sale

Reply:
If the LFC is only plunderable (or in "Korath Advanced") then it can actually be a small improvement - e.g. `"ramscoop" 1 / 1.25`.

Reply:
The LFC would be twice the size of the SFC but with .5/.2=2.5× its ramscoop.
I'm not sure yet if introducing a SFC would be a good idea though.

Reply:
FWIW, I would support removing the Fuel Pod for Korath outfitters once this is in.

Reply:
One second thought, if a SMC is to be introduced, it could be even smaller, e.g.:
* LFC: 20 mass, 200 capacity, 1 ramscoop
* SFC: 6 mass, 50 capacity, 0.25 ramscoop

So one LFC would be four times as effective as the SFC but more compact (20 instead of 4*6=24). However, I don't think introducing a SFC is necessary (a jump drive requires 200 fuel per jump, so why would the Korath producing anything with a smaller capacity?).